### PR TITLE
fix: alliance amino signatures

### PR DESCRIPTION
**Pull request by emidev98**
This pull request implements the following changes from alliance module https://github.com/terra-money/alliance/pull/173.

### CODE REVIEW

**Review by gregnuj — COMMENTED** [review]
Is this going to break consensus?

**Review by javiersuweijie — APPROVED** [review]
LGTM but we need to have a proper release plan for this since this is a consensus breaking change

**Comment by javiersuweijie** [comment]
@gregnuj should this PR be merged into the `release/2.3` branch instead?

File: go.mod [hunk 1]
```diff
@@ -18,10 +18,10 @@ require (
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/viper v1.15.0
 	github.com/strangelove-ventures/packet-forward-middleware/v6 v6.0.2
-	github.com/stretchr/testify v1.8.1
+	github.com/stretchr/testify v1.8.2
 	github.com/tendermint/tendermint v0.34.27
 	github.com/tendermint/tm-db v0.6.8-0.20221109095132-774cdfe7e6b0
-	github.com/terra-money/alliance v0.1.0-beta1
+	github.com/terra-money/alliance v0.1.1-rc.0
```
Comment (javiersuweijie) [inline go.mod:24]: Can we update to use `[v0.1.1](https://github.com/terra-money/alliance/releases/tag/v0.1.1)`

**Review by gregnuj — APPROVED** [review]

**Comment by gregnuj** [comment]
merge to main.... will be release 2.4 since it breaks consensus

**Pull request merged**